CrawlerTests/Data/ToBeForms.csv

```diff
@@ -0,0 +1,8 @@
+am
+is
+are
+was
+were
+being
+been
+be
```

Comment:
why is it here? you can use mock words even use something without and meaning it's too much overhead for the tests

Reply:
I implemented it using a text file for uniformity with the other words files (emotions, questions and numbers).

Reply:
Yes but you dont have to use real words, so when you add new words it won't affect the tests